src/main/java/net/pms/dlna/DLNAImageProfile.java

```diff
@@ -1256,22 +1258,10 @@ public DLNAComplianceResult checkCompliance(ImageInfo imageInfo) {
 		}
 
 		switch (this.toInt()) {
-			case DLNAImageProfile.GIF_LRG_INT:
-				checkGIF(imageInfo, complianceResult);
-				break;
-			case DLNAImageProfile.JPEG_LRG_INT:
-			case DLNAImageProfile.JPEG_MED_INT:
-			case DLNAImageProfile.JPEG_RES_H_V_INT:
-			case DLNAImageProfile.JPEG_SM_INT:
-			case DLNAImageProfile.JPEG_TN_INT:
-				checkJPEG(imageInfo, complianceResult);
-				break;
-			case DLNAImageProfile.PNG_LRG_INT:
-			case DLNAImageProfile.PNG_TN_INT:
-				checkPNG(imageInfo, complianceResult);
-				break;
-			default:
-				throw new IllegalStateException("Illegal DLNA media profile");
+			case DLNAImageProfile.GIF_LRG_INT -> checkGIF(imageInfo, complianceResult);
+			case DLNAImageProfile.JPEG_LRG_INT, DLNAImageProfile.JPEG_MED_INT, DLNAImageProfile.JPEG_RES_H_V_INT, DLNAImageProfile.JPEG_SM_INT, DLNAImageProfile.JPEG_TN_INT -> checkJPEG(imageInfo, complianceResult);
```

Comment:
Some of these lines are getting long with this style of switch statement. I find the old style easier to read, with one case per line. Does anyone else have an opinion @UniversalMediaServer/developers ?

Reply:
I modified it like that for readability :

```
		case DLNAImageProfile.GIF_LRG_INT -> checkGIF(imageInfo, complianceResult);
			case DLNAImageProfile.JPEG_LRG_INT,
				DLNAImageProfile.JPEG_MED_INT,
				DLNAImageProfile.JPEG_RES_H_V_INT,
				DLNAImageProfile.JPEG_SM_INT,
				DLNAImageProfile.JPEG_TN_INT -> checkJPEG(imageInfo, complianceResult);
			case DLNAImageProfile.PNG_LRG_INT,
				DLNAImageProfile.PNG_TN_INT -> checkPNG(imageInfo, complianceResult);
			default -> throw new IllegalStateException("Illegal DLNA media profile");
```